app/plugins/cookies.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
         if (request.response.variety === 'view' && statusCode !== 404 && statusCode !== 500 && request.response.source.manager._context) {
           const cookiesPolicy = getCurrentPolicy(request, h)
           request.response.source.manager._context.cookiesPolicy = cookiesPolicy
+          request.response.source.manager._context.user = request.auth?.credentials?.account
```

Comment:
Not to sure why we are storing the user in a cookie?

Reply:
Not storing in cookie. Just adding name into context so that nunjuck can use that value to show who is logged in.